src/borg/testsuite/archiver.py

```diff
@@ -988,6 +988,7 @@ def test_readonly_repository(self):
             # Restore permissions so shutil.rmtree is able to delete it
             os.system('chmod -R u+w ' + self.repository_path)
 
+    @pytest.mark.skipif('BORG_TESTS_UNRELIABLE_MODES' in os.environ, reason='modes unreliable')
```

Comment:
I'm trying to indicate that the modes themselves are unreliable, not the tests on them. Maybe `BORG_TESTS_IGNORE_MODES`?